src/main/java/com/rackspace/salus/monitor_management/services/MonitorManagement.java

```diff
@@ -2318,11 +2318,11 @@ public RenderedMonitorTemplate renderMonitorTemplate(UUID monitorId, String reso
                 monitor.getContent(), resource));
       }
     } else {
-      log.error("Failed to find resourceId={} during processing of monitor={}",
-          resourceId, monitor);
+      log.info("Invalid resourceId={} provided when rendering monitorId={} template for tenantId={}",
```

Comment:
I think it was good as an error
```suggestion
      log.error("Invalid resourceId={} provided when rendering monitorId={} template for tenantId={}",
```

Reply:
@itzg I suggested it be `info`.  It's an error to the customer making a request but not to us.  The system processed things as expected.

Reply:
Sorry, I missed that and that makes sense.